[2023-04-09T00:34:57.446+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:34:55.205875+00:00 [queued]>
[2023-04-09T00:34:57.453+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:34:55.205875+00:00 [queued]>
[2023-04-09T00:34:57.454+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T00:34:57.455+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-04-09T00:34:57.456+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T00:34:57.463+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-04-09 00:34:55.205875+00:00
[2023-04-09T00:34:57.469+0000] {standard_task_runner.py:55} INFO - Started process 9811 to run task
[2023-04-09T00:34:57.473+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'api_dag', 'is_api_available', 'manual__2023-04-09T00:34:55.205875+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/api.py', '--cfg-path', '/tmp/tmp2h8flrxt']
[2023-04-09T00:34:57.476+0000] {standard_task_runner.py:83} INFO - Job 72: Subtask is_api_available
[2023-04-09T00:34:57.533+0000] {task_command.py:376} INFO - Running <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:34:55.205875+00:00 [running]> on host c12a518d534c
[2023-04-09T00:34:57.586+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=api_dag
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-04-09T00:34:55.205875+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-09T00:34:55.205875+00:00
[2023-04-09T00:34:57.589+0000] {http.py:120} INFO - Poking: api/
[2023-04-09T00:34:57.599+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-04-09T00:34:57.602+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-04-09T00:34:57.813+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-04-09T00:34:57.834+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=api_dag, task_id=is_api_available, execution_date=20230409T003455, start_date=20230409T003457, end_date=20230409T003457
[2023-04-09T00:34:57.890+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-09T00:34:57.924+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
